plonky2/src/util/reducing.rs

```diff
@@ -82,15 +84,15 @@ impl<F: Field> ReducingFactor<F> {
 
     pub fn reduce_polys_base<BF: Extendable<D, Extension = F>, const D: usize>(
         &mut self,
-        polys: impl IntoIterator<Item = impl Borrow<PolynomialCoeffs<BF>>>,
+        polys: Vec<&PolynomialCoeffs<BF>>,
```

Comment:
Could this take something like a `impl MaybeParIter`, to avoid collecting?

Reply:
I'm not really able to make it compile without a Vec. Tried a bunch of variations...
Anyway the issue with collect is fixed.